README.md

```diff
@@ -47,7 +47,8 @@ JSNativeSerializer.stringify(arr); // returns "[1,2,3]"
 ```
 If it doesn't know how to serialize input it'll be serialized as `undefined`
 
-2) To simple js mirrored hierarchy in java with `JSParser` (hierarchy has `JSValue` as superclass)
+2) To simple js mirrored hierarchy in java with `JSParser` (hierarchy has
```

Comment:
This should've been in a different pr or commit as they are not related to this one. But let's leave as it is now.